course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -1,13 +1,44 @@
+// regex taken from https://www.ietf.org/rfc/rfc3986.txt appendix B, which
```

Comment:
That's pretty old, although of course the def of a URI is also old. :) Is there any way to tell if there's another more recent RFC or accepted recommendation that builds on or fixes any bugs in this regex?